blazor/dropdown-list/templates.md

```diff
@@ -7,307 +7,107 @@ control: DropDown List
-    <DropDownListFieldSettings Text="FirstName" Value="Country"></DropDownListFieldSettings>
-</SfDropDownList>
-
-@code {
-    public class EmployeeData
-    {
-        public string FirstName { get; set; }
-        public string Country { get; set; }
-    }
-    public EmployeeData Data = new EmployeeData();
-    public Query Query = new Query();
-}
-
-<style>
-    .country {
-        right: 15px;
-        position: absolute;
-    }
-</style>
-```
-
-
-
-![Blazor DropDownList with ItemTemplate](./images/blazor-dropdownlist-item-template.png)
+{% highlight cshtml %}
 
-## Value template
+{% include_relative code-snippet/template/value-template.razor %}
 
-The currently selected value that is displayed by default on the DropDownList input element can be customized using the [ValueTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_ValueTemplate) property.
+{% endhighlight %}
 
-In the following sample, the selected value is displayed as a combined text of both `FirstName` and `Designation` in the DropDownList input, which is separated by a hyphen.
+![Blazor DropDownList with ValueTemplate](./images/template/blazor-dropdownlist-value-template.png)
 
-```cshtml
-@using Syncfusion.Blazor.Data
-@using Syncfusion.Blazor.DropDowns
-
-<SfDropDownList TValue="string" TItem="EmployeeData" Placeholder="Select a customer" Query="@Query">
-    <DropDownListTemplates TItem="EmployeeData">
-        <ItemTemplate>
-            <span><span class='name'>@((context as EmployeeData).FirstName)</span><span class='destination'>@((context as EmployeeData).Designation)</span></span>
-        </ItemTemplate>
-        <ValueTemplate>
-            <span>@((context as EmployeeData).FirstName) - @((context as EmployeeData).Designation)</span>
-        </ValueTemplate>
-    </DropDownListTemplates>
-    <SfDataManager Url="https://ej2services.syncfusion.com/production/web-services/api/Employees" Adaptor="Syncfusion.Blazor.Adaptors.WebApiAdaptor" CrossDomain="true"></SfDataManager>
-    <DropDownListFieldSettings Text="FirstName" Value="Designation"></DropDownListFieldSettings>
-</SfDropDownList>
-
-@code {
-    public class EmployeeData
-    {
-        public string FirstName { get; set; }
-        public string Designation { get; set; }
-    }
-    public Query Query = new Query();
-}
-
-<style>
-    .destination {
-        right: 15px;
-        position: absolute;
-    }
-</style>
-```
-
-
-
-![Blazor DropDownList with ValueTemplate](./images/blazor-dropdownlist-value-template.png)
+## Item template
 
-## Group template
+The content of each list item within the DropDownList can be customized with the help of [ItemTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_ItemTemplate) property.
```

Comment:
The content of each list item within the DropDownList can be customized with the help of the [ItemTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_ItemTemplate) property.